server/handlers/component_handler.go

```diff
@@ -219,6 +219,9 @@ func (h *Handler) GetMeshmodelModels(rw http.ResponseWriter, r *http.Request) {
 		OrderOn:     queryParams.Get("order"),
 		Sort:        queryParams.Get("sort"),
 		Annotations: returnAnnotationComp,
+
+		Components:    queryParams.Get("components") == "true",
+		Relationships: queryParams.Get("relationships") == "true",
```

Comment:
a couple of lint checks to look into, @VihasMakwana

Reply:
yup. will push in a moment

Reply:
@leecalcote these are due to undefined references in meshkit.
Once I get https://github.com/meshery/meshkit/pull/428 merged, all these lints will be green.
Let's keep this PR open until we merge the former.

Reply:
Oh, gotcha. That makes sense.